.gitattributes

```diff
@@ -0,0 +1,11 @@
+/examples/ export-ignore
+/appveyor.yml export-ignore
+/panther.png export-ignore
+/.php_cs.dist export-ignore
+/phpstan.neon export-ignore
+/phpunit.xml.dist export-ignore
+/tests/ export-ignore
+/.travis.yml export-ignore 
+/.gitignore export-ignore
+/.gitattributes export-ignore
+/README.md export-ignore
```

Comment:
We should kip it in the generated archive don't you think?

Reply:
Do you mean the README.md?

Reply:
yes

Reply:
If you prefer, I can remove the README from the ignored files